components/mobile-scroll-lock/README.md

```diff
@@ -0,0 +1,21 @@
+MobileScrollLock
+================
+
+MobileScrollLock is a content-free React component for declaratively preventing scroll bleed from modal UI to the page body. This component applies a `lockscroll` class to the `document.documentElement` and `document.scrollingElement` elements to stop the body from scrolling. When it is present, the lock is applied.
+
+## Usage
+
+Declare scroll locking as part of modal UI.
+
+```jsx
+import { MobileScrollLock } from '@wordpress/components';
+
+function Sidebar( { isMobile } ) {
```

Comment:
I still think that using HOC would be the cleanest solution because the scroll lock behavior would be applied on top of the component that needs it.

```js
function Sidebar() {
	return (
		<div>
			Sidebar Content!
		</div>
	);
}
export default withMobileScrollLock( Sidebar );
```